docs/pages/server-access/guides/vscode.mdx

```diff
@@ -93,7 +93,14 @@ A new "Window Indicator" (icon with two arrows) should appear in the bottom left
 ![Window Indicator in bottom left corner of VS Code](../../../img/vscode/window-indicator.png)
 </Figure>
 
-Click on it, and select "Connect to Host...".
+Prior to connecting with a host set the `Remote.SSH: Use Local Server` setting to false in
+the extension setting.
+
+<Figure align="left" bordered caption="VSCode Setting">
+![Window Indicator in bottom left corner of VS Code](../../../img/vscode/settings.png)
+</Figure>
+
+To connect click on the Icon with two arrows, and select "Connect to Host...".
```

Comment:
```suggestion
To connect, click on the icon with two arrows and select "Connect to Host...".
```
Or should this be "Window Indicator", since we introduced that earlier in this section?